cli/src/main/java/org/jboss/as/cli/util/CLIExpressionResolver.java

```diff
@@ -48,6 +49,9 @@
     /** Environment variable prefix */
     private static final String ENV_PREFIX = "env.";
 
+    /** Security Vault pattern */
+    private static final Pattern VAULT_PATTERN = Pattern.compile("VAULT::.*::.*::.*");
```

Comment:
This pattern comes from: https://github.com/wildfly/wildfly-core/blob/master/server/src/main/java/org/jboss/as/server/services/security/RuntimeVaultReader.java#L53